docs/_posts/2017-09-26-react-v16.0.md

```diff
@@ -0,0 +1,133 @@
+---
+title: "React v16.0"
+author: [acdlite]
+---
+
+We're excited to announce the release of React v16.0! Among the changes are some long-standing feature requests, including [**fragments**](#new-render-return-types-fragments-and-strings), [**error boundaries**](#better-error-handling), [**portals**](#portals), support for [**custom DOM attributes**](#support-for-custom-dom-attributes), improved [**server-side rendering**](#better-server-side-rendering), and [**reduced file size**](#reduced-file-size).
+
+### New `render` return types: fragments and strings
+
+You can now return an array of elements from a component's `render` method:
+
+```js
+render() {
+  // No need to wrap list items in an extra element!
+  return [
+    // Don't forget the keys :)
+    <li key="A"/>First item</li>,
+    <li key="B"/>Second item</li>,
+    <li key="C"/>Third item</li>,
+  ];
+}
+```
+
+Support for returning strings from the render method has also been added.
+
+[API Documentation](/docs/react-component.html#render)
+
+### Better error handling
+
+
+Previously, runtime errors during rendering could potentially put React in a broken state, producing cryptic error messages and requiring a page refresh to recover. To address this problem, React 16 uses a more resilient error-handling strategy. By default, if an error is thrown inside a component's render or lifecycle methods, the whole component tree is unmounted from the root. This prevents the display of corrupted data. However, it's probably not the ideal user experience.
+
+Instead of unmounting the whole app every time there's an error, you can use error boundaries. Error boundaries are special components that capture errors inside their subtree and display a fallback UI in its place. Think of error boundaries like try-catch statements, but for React components.
+
+For more details, check out our [previous post on error handling in React 16](/blog/2017/07/26/error-handling-in-react-16.html).
+
+
+### Portals
+
+Portals provide a first-class way to render children into a DOM node that exists outside the DOM hierarchy of the parent component.
+
+```js{6}
+render() {
+  // React does *not* create a new div. It renders the children into `divNode`.
+  // `divNode` is any valid DOM node, regardless of its location in the DOM.
+  return React.createPortal(
```

Comment:
It's on `ReactDOM`. (Although we might provide an isomorphic version someday.)

Reply:
came here to comment this 😅